### ISSUE TITLE
Замена программного плеера на ss iptv или подобный 

### ISSUE THREAD

**Issue by svkomarov** [yumata/lampa#14]
Для поддержки сабов, выбора аудиодорожки и тд

**Comment by yumata** [comment]
Это сторонний виджет, как я вам его сделаю? :D

**Comment by svkomarov** [comment]
Криво выразился) у ss iptv вроде есть документация по их плееру, может что-то удастся использовать

**Comment by yumata** [comment]
Верно, это ихний плеер и документация к ихнему плееру, другие лижут лапу.

**Comment by qarkai** [comment]
> у ss iptv вроде есть документация по их плееру

Дайте, пожалуйста, ссылку.
